Mathlib/Geometry/Manifold/ChartedSpace.lean

```diff
@@ -862,26 +862,24 @@ instance ChartedSpace.sum : ChartedSpace H (M ⊕ M') where
   chartAt := Sum.elim (fun x ↦ (cm.chartAt x).lift_openEmbedding IsOpenEmbedding.inl)
     (fun x ↦ (cm'.chartAt x).lift_openEmbedding IsOpenEmbedding.inr)
   mem_chart_source p := by
-    by_cases h : Sum.isLeft p
-    · let x := Sum.getLeft p h
-      rw [Sum.eq_left_getLeft_of_isLeft h, Sum.elim_inl, lift_openEmbedding_source,
+    cases p with
```

Comment:
Thanks for teaching me this trick! My next PRs will use this from the beginning.